### DISCUSSION TITLE
Support for Pydantic v2

### DISCUSSION THREAD

**Discussion by farzbood** [tortoise/tortoise-orm#1436] [Polls]
Hi everyone,
Since **FastAPI (0.100.0**) released with support for **Pydantic v2**, which brings big improvments in app's performance, how about a version bump in **TortoiseORM** to be compatible with **Pydantic v2** specifications?
This is to call for this upgrade/bump to support **FastAPI 0.100.0** and **Pydantic v2**.
Happy development, PEACE V ;)

**Comment by long2ice** [comment]
See https://github.com/tortoise/tortoise-orm/pull/1427

**Reply by farzbood** [reply]
Good luck with finalizing the upgrade, looking forward the new version release.
Keep up with outstanding work.

P.S
I've searched for the topic before starting this poll, but didn't get any results!
